src/drawing/backend_impl/piston.rs

```diff
@@ -34,6 +34,14 @@ fn make_point_pair(a: BackendCoord, b: BackendCoord, scale: f64) -> [f64; 4] {
     ]
 }
 
+fn make_circle(center: BackendCoord, radius: u32, scale: f64) -> [f64; 4] {
```

Comment:
To me this isn't related to the PR, right?

Reply:
I think @pelekhay added this commit

Reply:
So would you mind rebase to the master branch - since you are actually based on the 0.2 branch

Reply:
ah my bad, I will rebase